tensornetwork/matrixproductstates/base_mps_test.py

```diff
@@ -334,43 +316,43 @@ def test_measure_two_body_correlator_value_error(backend):
   tensors = 6 * [backend.convert_to_tensor(tensor)]
   operator = backend.convert_to_tensor(
       np.array([[1, -1], [-1, 1]], dtype=np.float64))
-  mps = BaseMPS(tensors, backend=backend)
+  mps = BaseMPS(tensors, backend=backend.name)
   with pytest.raises(ValueError):
     mps.measure_two_body_correlator(
         op1=operator, op2=operator, site1=-1, sites2=[2])
 
 
-def test_get_node(backend):
+def test_get_tensor(backend):
   backend = backend_factory.get_backend(backend)
   tensor1 = np.ones((2, 3, 2), dtype=np.float64)
   tensor2 = 2 * np.ones((2, 3, 2), dtype=np.float64)
   tensors = [tensor1, tensor2]
-  mps = BaseMPS(tensors, backend=backend)
-  np.testing.assert_allclose(mps.get_node(0).tensor, tensor1)
-  np.testing.assert_allclose(mps.get_node(1).tensor, tensor2)
+  mps = BaseMPS(tensors, backend=backend.name)
+  np.testing.assert_allclose(mps.get_tensor(0), tensor1)
+  np.testing.assert_allclose(mps.get_tensor(1), tensor2)
 
 
-def test_get_node_connector_matrix(backend):
+def test_get_tensor_connector_matrix(backend):
   backend = backend_factory.get_backend(backend)
   tensor1 = np.ones((2, 3, 2), dtype=np.float64)
   tensor2 = 2 * np.ones((2, 3, 2), dtype=np.float64)
   connector = backend.convert_to_tensor(np.ones((2, 2), dtype=np.float64))
   tensors = [tensor1, tensor2]
-  mps = BaseMPS(tensors, backend=backend, connector_matrix=connector)
-  np.testing.assert_allclose(mps.get_node(0).tensor, tensor1)
-  np.testing.assert_allclose(mps.get_node(1).tensor, 2 * tensor2)
+  mps = BaseMPS(tensors, backend=backend.name, connector_matrix=connector)
+  np.testing.assert_allclose(mps.get_tensor(0), tensor1)
+  np.testing.assert_allclose(mps.get_tensor(1), 2 * tensor2)
 
 
-def test_get_node_raises_error(backend):
+def test_get_tensor_raises_error(backend):
   backend = backend_factory.get_backend(backend)
   tensor1 = np.ones((2, 3, 2), dtype=np.float64)
   tensor2 = 2 * np.ones((2, 3, 2), dtype=np.float64)
   tensors = [tensor1, tensor2]
-  mps = BaseMPS(tensors, backend=backend)
+  mps = BaseMPS(tensors, backend=backend.name)
```

Comment:
You can remove all of the `.name` additions if you still allow backend objects to be passed.